updatecli/updatecli.d/nodejs.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: Bump `nodejs` version according to packer-images
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastPackerImageVersion:
+    kind: shell
+    name: Get the latest `nodejs_linux_version` from https://github.com/jenkins-infra/packer-images/blob/main/provisioning/tools-versions.yml
+    spec:
+      command: |
+        bash -c 'curl --silent --fail --show-error "https://raw.githubusercontent.com/jenkins-infra/packer-images/main/provisioning/tools-versions.yml" | grep nodejs_linux_version | sed "s/.*: //"'
```

Comment:
- Blocking: you must use the NodeJS version of the production-deployed Packer image. Checking for the `main` branch is an error which can lead to breaking the project here: we can have hours, even days between a version on the `main` branch and the version on production for infra.ci and ci.jio. Look at [Maven version manifest on Puppet](https://github.com/jenkins-infra/jenkins-infra/blob/31613da9c34f819ed91708155087a8581dd30e35/updatecli/weekly.d/jenkinscontroller-tools-maven.yaml#L16-L36) where we use 2 sources: one to retrieve the production version, and the second to get the Maven version from this "production" image.
- Nit: the shell script here does the job 👍 But you can use an updatecli resource of kind `yaml` and parse it on the go: way more portable. I just realized this for [Maven version manifest on Puppet](https://github.com/jenkins-infra/jenkins-infra/blob/31613da9c34f819ed91708155087a8581dd30e35/updatecli/weekly.d/jenkinscontroller-tools-maven.yaml#L16-L36) so I'm gonna try to update it (instead of the ugly file resource with transformer). Benefit: using native YAML parsing allows portability and increase readability.

Reply:
(see for the nitpick: https://github.com/jenkins-infra/jenkins-infra/pull/3483)